sql/core/src/test/scala/org/apache/spark/sql/internal/SQLConfSuite.scala

```diff
@@ -310,4 +310,10 @@ class SQLConfSuite extends QueryTest with SharedSQLContext {
     SQLConf.unregister(fallback)
   }
 
+  test("[SPARK-24783][SQL]spark.sql.shuffle.partitions=0 should throw exception ") {
```

Comment:
nit: let's fix the test title to be consistent: `test("SPARK-24783: spark.sql.shuffle.partitions=0 should throw exception") {`